features/org.eclipse.emf.mwe2.launcher-feature/feature.xml

```diff
@@ -36,7 +36,7 @@ SPDX-License-Identifier: EPL-2.0
       <import plugin="org.eclipse.xtext.common.types" version="2.12.1" match="greaterOrEqual"/>
       <import plugin="org.eclipse.xtext.xbase" version="2.12.1" match="greaterOrEqual"/>
```

Comment:
2.12.1 sounds wired here

Reply:
Should be `2.25.0`, right?

Reply:
think so

Reply:
is it our version adapt script that sets this wired version?

Reply:
I added this change to the PR.